geometry/graph/offset.test.js

```diff
@@ -26,78 +26,46 @@ test('offset', (t) => {
     testGraph(geometry);
   }
   t.deepEqual(offsetGeometries.map(serialized), [
-    '36\n' +
-      '-13/19 -15/26 1/2\n' +
-      '-2/3 -11/18 1/2\n' +
-      '-16/23 -7/13 1/2\n' +
-      '-7/13 -16/23 1/2\n' +
-      '-1/2 -7/10 1/2\n' +
-      '-15/26 -13/19 1/2\n' +
-      '-25/39 -25/39 1/2\n' +
-      '-11/18 -2/3 1/2\n' +
-      '-7/10 -1/2 1/2\n' +
-      '-7/10 1/2 1/2\n' +
-      '1/2 -7/10 1/2\n' +
-      '7/13 -16/23 1/2\n' +
-      '15/26 -13/19 1/2\n' +
-      '11/18 -2/3 1/2\n' +
-      '25/39 -25/39 1/2\n' +
-      '13/19 -15/26 1/2\n' +
-      '2/3 -11/18 1/2\n' +
-      '16/23 -7/13 1/2\n' +
-      '7/10 -1/2 1/2\n' +
-      '7/10 1/2 1/2\n' +
-      '1/2 7/10 1/2\n' +
-      '16/23 7/13 1/2\n' +
-      '13/19 15/26 1/2\n' +
-      '2/3 11/18 1/2\n' +
-      '15/26 13/19 1/2\n' +
-      '7/13 16/23 1/2\n' +
-      '11/18 2/3 1/2\n' +
-      '25/39 25/39 1/2\n' +
-      '-1/2 7/10 1/2\n' +
-      '-7/13 16/23 1/2\n' +
-      '-11/18 2/3 1/2\n' +
-      '-25/39 25/39 1/2\n' +
-      '-15/26 13/19 1/2\n' +
-      '-2/3 11/18 1/2\n' +
-      '-16/23 7/13 1/2\n' +
-      '-13/19 15/26 1/2\n' +
+    '20\n' +
+      '-41/64 -41/64 1/2\n' +
+      '-3152519739159347/4503599627370496 -1/2 1/2\n' +
+      '-385724968084973/562949953421312 -2598230554252209/4503599627370496 1/2\n' +
```

Comment:
Why did these ones seemingly get more complex instead of less?

Reply:
Hmm, that's a good question.